Build fails due to duplicate methods

src/github.com/go-swagger/go-swagger/scan/parameters.go:75:6: paramTypable.WithEnum redeclared in this block
	previous declaration at src/github.com/go-swagger/go-swagger/scan/parameters.go:43:6
src/github.com/go-swagger/go-swagger/scan/parameters.go:110:6: itemsTypable.WithEnum redeclared in this block
	previous declaration at src/github.com/go-swagger/go-swagger/scan/parameters.go:94:6
src/github.com/go-swagger/go-swagger/scan/responses.go:97:6: responseTypable.WithEnum redeclared in this block
	previous declaration at src/github.com/go-swagger/go-swagger/scan/responses.go:41:6
src/github.com/go-swagger/go-swagger/scan/scanner.go:477:2: duplicate method WithEnum
src/github.com/go-swagger/go-swagger/scan/schema.go:1047:8: ambiguous selector prop.WithEnum
The command '/bin/sh -c go get github.com/go-swagger/go-swagger/cmd/swagger' returned a non-zero code: 2
The command "docker build -f Dockerfile -t $DOCKER_IMAGE_NAME:$COMMIT ." failed and exited with 2 during .


TravisCLI
go version: 1.10.3  
OS: linux/amd64

--- a/scan/parameters.go
+++ b/scan/parameters.go
@@ -41,7 +41,7 @@ func (pt paramTypable) Typed(tpe, format string) {
 }
 
 func (pt paramTypable) WithEnum(values ...interface{}) {
-	pt.param.WithEnum(values...);
+	pt.param.WithEnum(values...)
 }
 
 func (pt paramTypable) SetRef(ref spec.Ref) {
@@ -72,10 +72,6 @@ func (pt paramTypable) Schema() *spec.Schema {
 	return pt.param.Schema
 }
 
-func (pt paramTypable) WithEnum(values ...interface{}) {
-	pt.param.WithEnum(values...)
-}
-
 type itemsTypable struct {
 	items *spec.Items
 	level int
@@ -92,7 +88,7 @@ func (pt itemsTypable) SetRef(ref spec.Ref) {
 }
 
 func (pt itemsTypable) WithEnum(values ...interface{}) {
-	pt.items.WithEnum(values...);
+	pt.items.WithEnum(values...)
 }
 
 func (pt itemsTypable) Schema() *spec.Schema {
@@ -105,10 +101,6 @@ func (pt itemsTypable) Items() swaggerTypable {
 	}
 	pt.items.Type = "array"
 	return itemsTypable{pt.items.Items, pt.level + 1}
-}
-
-func (pt itemsTypable) WithEnum(values ...interface{}) {
-	pt.items.WithEnum(values...)
 }
 
 type paramValidations struct {

--- a/scan/responses.go
+++ b/scan/responses.go
@@ -94,10 +94,6 @@ func (ht responseTypable) CollectionOf(items *spec.Items, format string) {
 	ht.header.CollectionOf(items, format)
 }
 
-func (ht responseTypable) WithEnum(values ...interface{}) {
-	ht.header.WithEnum(values)
-}
-
 type headerValidations struct {
 	current *spec.Header
 }

--- a/scan/scanner.go
+++ b/scan/scanner.go
@@ -474,7 +474,6 @@ type swaggerTypable interface {
 	WithEnum(...interface{})
 	Schema() *spec.Schema
 	Level() int
-	WithEnum(...interface{})
 }
 
 // Map all Go builtin types that have Json representation to Swagger/Json types.

